.github/workflows/test-environment.yml

```diff
@@ -0,0 +1,223 @@
+name: Sanity tests
+
+on:
+  # Ability to execute on demand
+  workflow_dispatch:
+    inputs:
+      deployment_name:
+        type: string
+        description: |
+          Name your environment (Only a-zA-Z0-9 and `-`).
+          For example: john-8-7-2-June01'
+        required: true
+      ec-api-key:
+        type: string
+        description: "Elastic Cloud API KEY"
```

Comment:
Can we have hyperlinks here? Getting the API is not so obvious in the cloud UI, maybe we can attach the relevant link (https://cloud.elastic.co/deployment-features/keys)

Reply:
I've attempted to add a hyperlink, but it is displayed as plain text instead. As a result, I have made updates to the documentation in the README file to provide detailed instructions on how to generate an Elastic Cloud API key.